topics/microbiome/tutorials/multivariable-association/tutorial.md

```diff
@@ -0,0 +1,478 @@
+---
+layout: tutorial_hands_on
+
+title: "Determining multivariable association between various meta’omic features using MaAslin2"
```

Comment:
```suggestion
title: "Determining multivariable association of various meta’omic features using MaAslin2"
```
`Between` would mean that we determine the associations between feature (but we do between meta'omics and the metadata)'

Reply:
Thanks for pointing it out :)